src/otx/recipe/instance_segmentation/rtmdet_inst_tiny.yaml

```diff
@@ -0,0 +1,146 @@
+model:
+  class_path: otx.algo.instance_segmentation.rtmdet_inst.RTMDetInst
+  init_args:
+    num_classes: 80
+    variant: tiny
+
+optimizer:
+  class_path: torch.optim.AdamW
+  init_args:
+    lr: 0.004
+    weight_decay: 0.05
+
+scheduler:
+  - class_path: otx.algo.schedulers.warmup_schedulers.LinearWarmupScheduler
+    init_args:
+      num_warmup_steps: 20
+  - class_path: lightning.pytorch.cli.ReduceLROnPlateau
+    init_args:
+      mode: max
+      factor: 0.1
+      patience: 10
+      monitor: val/map_50
+      min_lr: 4e-06
+
+metric:
+  class_path: otx.algo.instance_segmentation.otx_instseg_evaluation.OTXMaskRLEMeanAveragePrecision
+  init_args:
+    box_format: xyxy
+    iou_type: segm
+
+engine:
+  task: INSTANCE_SEGMENTATION
+  device: auto
+
+callback_monitor: val/map_50
+
+data: ../_base_/data/mmdet_base.yaml
+overrides:
+  precision: 32 # 16/"16-true" does not work
+  max_epochs: 100
+  gradient_clip_val: 35.0
+  data:
+    task: INSTANCE_SEGMENTATION
+    config:
+      include_polygons: true
+      train_subset:
+        batch_size: 4
+        num_workers: 10
+        transforms:
+          - type: LoadImageFromFile
+            backend_args: null
+          - type: LoadAnnotations
+            with_bbox: true
+            with_mask: true
+          - type: CachedMosaic
+            img_scale:
+              - 640
+              - 640
+            pad_val: 114.0
+            max_cached_images: 20
+            random_pop: false
+          - type: RandomResize
+            scale:
+              - 1280
+              - 1280
+            ratio_range:
+              - 0.5
+              - 2.0
+            keep_ratio: true
+          - type: RandomCrop
+            crop_size:
+              - 640
+              - 640
+          - type: YOLOXHSVRandomAug
+          - type: RandomFlip
+            prob: 0.5
+          - type: Pad
+            pad_to_square: true
+            pad_val: 114
+          - type: CachedMixUp
+            img_scale:
+              - 640
+              - 640
+            ratio_range:
+              - 1.0
+              - 1.0
+            max_cached_images: 10
+            random_pop: false
+            pad_val:
+              - 114
```

Comment:
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/8c88f298-4ca7-407e-9a04-1182424b5c3e)
The default pixel pad value of `DetDataPreprocessor` is 0. Update the model configuration to align with this value, 114.

Reply:
@vinnamkim Padding to a square aspect ratio has already been applied before DetDataPreprocessor. Specifically, YOLOX and RTMDet use padding value of 114, and has the following aug:
```
Type: Pad
pad_to_square: true
pad_val: 114
```
![image](https://github.com/openvinotoolkit/training_extensions/assets/13793254/b632f9bc-a9e8-41f4-af21-6571adad102f)

Reply:
If each image in a batch has different size, `DetDataPreprocessor` will pad them with zero (default) to the maximal image size of them. I observed there is image size mismatch between samples in a batch of instance segmentation models while working on https://github.com/openvinotoolkit/training_extensions/pull/3005.

Reply:
I added pad_value to preprocessor to both yolox variants and RTMDetInst: https://github.com/openvinotoolkit/training_extensions/pull/3039/files#diff-caf5ba5b8d9c9b0ee78ccdb4dec72273f5a3b0d74913ca6d9b7c6fa9aa871faaR13